src/redux/types.ts

```diff
@@ -1,16 +1,32 @@
 export const REGISTER_EMPLOYER_REQUEST = 'REGISTER_EMPLOYER_REQUEST'
 export const REGISTER_EMPLOYER_SUCCESS = 'REGISTER_EMPLOYER_SUCCESS'
 export const REGISTER_EMPLOYER_FAIL = 'REGISTER_EMPLOYER_FAIL'
+export const REGISTER_JOBSEEKER_REQUEST = 'REGISTER_JOBSEEKER_REQUEST'
+export const REGISTER_JOBSEEKER_SUCCESS = 'REGISTER_JOBSEEKER_SUCCESS'
+export const REGISTER_JOBSEEKER_FAIL = 'REGISTER_JOBSEEKER_FAIL'
+export const LOGIN_EMPLOYER_REQUEST = 'LOGIN_EMPLOYER_REQUEST'
+export const LOGIN_EMPLOYER_SUCCESS = 'LOGIN_EMPLOYER_SUCCESS'
+export const LOGIN_EMPLOYER_FAIL = 'LOGIN_EMPLOYER_FAIL'
+export const LOGOUT_EMPLOYER = 'LOGOUT_EMPLOYER'
+export const LOGIN_JOBSEEKER_REQUEST = 'LOGIN_JOBSEEKER_REQUEST'
+export const LOGIN_JOBSEEKER_SUCCESS = 'LOGIN_JOBSEEKER_SUCCESS'
+export const LOGIN_JOBSEEKER_FAIL = 'LOGIN_JOBSEEKER_FAIL'
+export const LOGOUT_JOBSEEKER = 'LOGOUT_JOBSEEKER'
 
-export type employerActions =
+export type EmployerActions =
   | RegisterEmployerRequestAction
   | RegisterEmployerSuccessAction
   | RegisterEmployerFailAction
+  | LoginEmployerRequestAction
+  | LoginEmployerSuccessAction
+  | LoginEmployerFailAction
+  | LogoutEmployerAction
 
 export type RegisterEmployerRequestAction = {
   type: typeof REGISTER_EMPLOYER_REQUEST
   payload: {
-    credentials: {
+    info: object
```

Comment:
You should not give it `object` type since it does not provide any info about `info`, you could check what you get back from the api response and model the `info` type ater the response. Or for now you can just use any 😄

Reply:
Oh yes, this is some old code.